tests/helper/kubernetes_utils.go

```diff
@@ -2,11 +2,20 @@ package helper
 
 import (
 	"fmt"
+	"os"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+// CopyKubeConfigFile copies default kubeconfig file into current context config file
+func CopyKubeConfigFile(kubeConfigFile, tempConfigFile string, info os.FileInfo) string {
+	err := CopyFile(kubeConfigFile, tempConfigFile, info)
```

Comment:
change it as per https://github.com/openshift/odo/pull/2965/files#r415551858

Reply:
https://github.com/openshift/odo/pull/2965#discussion_r415560462